docs/tutorials/podman_tutorial.md

```diff
@@ -17,16 +17,22 @@ In this section, we will help you install all the runtime and build dependencies
 acquire the source, and build it.
 
 ### Installing build and runtime dependencies
-```
+```bash
 $ sudo dnf install -y git runc libassuan-devel golang golang-github-cpuguy83-go-md2man glibc-static \
                                     gpgme-devel glib2-devel device-mapper-devel libseccomp-devel \
                                     atomic-registries iptables skopeo-containers containernetworking-cni \
                                     conmon
 ```
 ### Building and installing podman
-```
-# git clone https://github.com/projectatomic/libpod/ ~/src/github.com/projectatomic/libpod
-# cd !$
+
+First, configure a `GOPATH` (if you are using go1.8 or later, this defaults to `~/go`), then clone
+and make libpod.
+
+```bash
+# export GOPATH=~/go
+# mkdir -p $GOPATH
+# git clone https://github.com/projectatomic/libpod/ $GOPATH/src/github.com/projectatomic/libpod
```

Comment:
Can't we leave `GOPATH` unset and use `go get -d github.com/projectatomic/libpod`?  Or, if we really need a `GOPATH`, could we use:

```console
$ export GOPATH="$(go env GOPATH)"
```

Also you're using `#` prompts for most of this, and `#` is traditionally root.  Can we use `$` (traditionally not root) to clone and build, and then either `# make install ...` or `$ sudo make install ...` below for the only step where we actually need root?

Reply:
> This note is primarily for new to golang folks...

Who can probably not bother setting `GOPATH` at all, right?  Just let Go fall back to whatever it thinks is a reasonable default.